zip-0316.rst

```diff
@@ -303,15 +303,40 @@ Open Issues and Known Concerns
 Privacy impacts of transparent or cross-pool transactions, and the
 associated UX issues, will be addressed in ZIP 315 (in preparation).
 
-Revision History
-================
+Specification
+=============
+
+Revisions
+---------
+
+.. _`Revision 0`:
+
+* Revision 0: The initial version of this specification.
 
 .. _`Revision 1`:
 
-* Revision 1: `MUST-understand Typecodes`_ and `Address Expiration Metadata`_
+* Revision 1: This version adds support for `MUST-understand Typecodes`_
+  and `Address Expiration Metadata`_
+
+A Revision 0 UA/UVK is distinguished from a Revision 1 UA/UVK by its
+Human-Readable Part, as follows.
+
+Let *prefix* be:
+* “``u``”, if this is a UA/UVK prior to `Revision 1`_;
+* “``ua``”, if this is a UA/UVK from `Revision 1`_ onward.
```

Comment:
This choice of prefix could be confusing given that it is also the prefix for Revision 1 UVKs.